posts/everything-you-need-to-know-about-pewdiepie-s-ai-setup/index.qmd

```diff
@@ -0,0 +1,209 @@
+- Speed: even a MacBook can answer faster than waiting for a web queue.
+- Play: Felix treats this like digital LEGO—swap models, personas, or tools until it fits how you think.
+
+Skim the sections that match your hardware, then follow the path that sounds fun.
+
+## Quick glossary
+
+- **Quant (quantization):** shrink a model so it fits in laptop memory—think of it as zipping weights.
+- **RAG (retrieval augmented generation):** let the model look up your files before it answers you.
+- **Tensor split:** share a big model across multiple GPUs so none of them overload.
+
+## Models Felix name‑dropped
+
+- [Meta-Llama-3.1-70B-Instruct](https://huggingface.co/meta-llama/Meta-Llama-3.1-70B-Instruct) – his first “it just works” checkpoint before chasing bigger beasts. Grab [mlx-community/Meta-Llama-3.1-70B-Instruct-4bit-mlx](https://huggingface.co/mlx-community/Meta-Llama-3.1-70B-Instruct-4bit-mlx) for Apple Silicon or any 70B [GGUF](https://huggingface.co/models?search=Meta-Llama-3.1-70B-Instruct+gguf) build for llama.cpp.
+- [openai/gpt-oss-120b](https://huggingface.co/openai/gpt-oss-120b) – the newly released 120B open model he runs through vLLM for fast chat. Community quantizations like [bartowski/GPT-OSS-120B-GGUF](https://huggingface.co/bartowski/GPT-OSS-120B-GGUF) make it llama.cpp-friendly.
+- [Qwen/Qwen3-235B-A22B-Instruct-2507](https://huggingface.co/Qwen/Qwen3-235B-A22B-Instruct-2507) – the “Chinese AI strikes again” giant he stretched to 100k tokens of context. If you’re on smaller gear, drop to [Qwen/Qwen2.5-7B-Instruct](https://huggingface.co/Qwen/Qwen2.5-7B-Instruct) or its [GGUF shelf](https://huggingface.co/models?search=Qwen2.5+GGUF) for the same toolchain with realistic VRAM demands.
+
+> “Can a guy just max out? Is that too much to ask?” was his mantra while he hunted for the biggest checkpoint that would still run. Use the table below to pick the right ceiling for your own machine before you start chasing 200B+ bragging rights.
+
+- For day-to-day tinkering you can downshift to [bartowski/GPT-OSS-20B-GGUF](https://huggingface.co/bartowski/GPT-OSS-20B-GGUF) and [Qwen/Qwen2.5-7B-Instruct-GGUF](https://huggingface.co/Qwen/Qwen2.5-7B-Instruct-GGUF); those are the examples below because they fit on almost any modern laptop when quantized.
+
+| Model (Quant) | Minimum unified memory (Mac) | Minimum GPU VRAM (PC/Linux) | Notes |
+| --- | --- | --- | --- |
+| Qwen2.5-7B Q4 (chat) | 12 GB | 8 GB | Runs comfortably on M2/M3 with 16 GB RAM or a 3060/4060 laptop GPU. |
+| GPT-OSS-20B Q4_K | 24 GB | 12 GB | Expect ~8 tokens/s on M3 Pro 18 GB or a desktop 4070. |
+| Llama‑3.1‑70B Q4_K | 64 GB | 48 GB | Needs M3 Ultra 64 GB or dual 3090/4090s split with tensor-split. |
+| GPT-OSS-120B Q4_K_M | 96 GB | 64 GB | Felix runs this on vLLM with four 4090s; single-node inference is tight otherwise. |
+| Qwen3-235B Q4_K_M | 128 GB+ | 96 GB+ | Practical only with 6–8 GPUs or Apple’s 192 GB Mac Pro; great demo, not daily driver. |
```

Comment:
* We could add links to the models.
* How about we add a table for llama.cpp (this one), and a separate one for MLX models?